generator/dashboards/operational_monitoring_dashboard.py

```diff
@@ -68,13 +79,17 @@ def to_lookml(self, bq_client, data):
                 f"/looker-hub/{self.namespace}/explores/{explore}.explore.lkml"
             )
 
+            series_colours = self._map_series_to_colours(
+                table_defn["branches"], explore
+            )
             for metric in metrics:
                 title = lookml_utils.slug_to_title(metric)
                 kwargs["elements"].append(
                     {
                         "title": title,
                         "metric": metric,
                         "explore": explore,
+                        "series_colours": series_colours,
```

Comment:
I think we should be consistent with `color` vs `colour`. Looker seems to be expecting `color`.

```suggestion
                       "series_colors": series_colors,
```